server/routes/api/documents/documents.ts

```diff
@@ -435,6 +436,49 @@ router.post(
   }
 );
 
+router.post(
+  "documents.users",
+  auth(),
+  validate(T.DocumentsUsersSchema),
+  async (ctx: APIContext<T.DocumentsUsersReq>) => {
+    const { id, query } = ctx.input.body;
+    const { user } = ctx.state.auth;
+    const document = await Document.findByPk(id);
+    authorize(user, "update", document);
+
+    let users: User[] = [];
+
+    if (document.collectionId) {
+      const collection = await Collection.findByPk(document.collectionId);
+      authorize(user, "update", collection);
+      const memberIds = await Collection.membershipUserIds(collection.id);
```

Comment:
This doesn't include the `createdBy` user of the document. Should we include it separately?

Reply:
I don't think so as they don't necessarily retain access to the document if it's been moved between collections or their permissions have changed

Reply:
Totally forgot we had this method